app/controllers/katello/api/v2/content_views_controller.rb

```diff
@@ -100,6 +100,7 @@ def publish
       task = async_task(::Actions::Katello::ContentView::Publish, @view, params[:description],
                         :major => params[:major],
                         :minor => params[:minor],
+                        :solve_dependencies => @view.solve_dependencies,
```

Comment:
Any reason to pass this in as a param rather than just using the view's value further down in the action?

Reply:
I can move it to `katello/app/lib/actions/katello/repository/clone_to_version.rb`, that seems to be the last action the CV is referenced

Reply:
sounds good!